src/Signer/Ecdsa/Sha384.php

```diff
@@ -18,8 +19,13 @@ public function getAlgorithmId(): string
     /**
      * {@inheritdoc}
      */
-    public function getAlgorithm(): string
+    public function getAlgorithm(): int
```

Comment:
BC break to be documented

Reply:
This method was always considered as internal, but we were not using `@internal` yet. I'm not sure if this really need to be stated as BC-break :(